README.md

```diff
@@ -14,10 +14,17 @@ and query service management code.
 
 ### Install dependencies
 
-`composer install`
+It is recommened to install the php dependencies via a composer docker container as that contains the required php extensions: 
+```
+docker run --rm -it -v $PWD:/app -u $(id -u):$(id -g) composer install
+```
+
 
-Alternatively you can use docker to run composer
-`docker run --rm --interactive --tty --volume $PWD:/app --user $(id -u):$(id -g) composer install`
+Alternatively, you can run `composer` on your local machine but make sure the `xml` and `curl` php extensions are installed. For example, on Ubuntu:
+```
+sudo apt install php-xml php-curl
+composer install
```

Comment:
```suggestion
composer install --ignore-platform-reqs
```

(Funny, I had to split this up into two comments, because apparently GH can't handle suggestions that include triple backticks <code>```</code>)

Reply:
Cool, thanks for pointing out that `composer` flag